cylc/uiserver/resolvers.py

```diff
@@ -322,11 +323,10 @@ async def play(
                 ret_code = proc.wait(timeout=120)
 
                 if ret_code:
-                    # command failed
+                    msg = f"Command failed ({ret_code}): {cmd_repr}"
                     out, err = proc.communicate()
-                    results[wflow] = err.strip() or out.strip() or (
-                        f'Command failed ({ret_code}): {cmd_repr}'
-                    )
+                    results[wflow] = err.strip() or out.strip() or msg
+                    log.error(f"{msg}\n{results[wflow]}")
```

Comment:
This could log `{msg}\n{msg}`?

Reply:
Yes, but unlikely as most of the time there should be stderr or stdout

Reply:
```suggestion
                    log.error(f"{msg}\n{err.strip() or out.strip()}")
```

?

Reply:
Or:

```suggestion
                    log.error(f"{msg}\nstdout: {out.strip()}\nstderr: {err.strip()}")
```